docs/docs/Quick_Start.md

```diff
@@ -6,15 +6,15 @@ description: >
     "Quick start guide"
 ---
 
-# Quick Start Guide for RedisBloom
+# Quick Start Guide for Redis Probabilistic
 
 ## Redis Cloud
 
-RedisBloom is available on all Redis Cloud managed services.  Redis Cloud Essentials offers a completely free managed database up to 30MB.
+Probabilistic is available on all Redis Cloud managed services.  Redis Cloud Fixed offers a completely free managed database up to 30MB.
 
 [Get started here](https://redis.com/try-free/)
 
-## Launch RedisBloom with Docker
+## Launch Probabilistic with Docker
```

Comment:
Launch Redis Stack with Docker

Reply:
Change made. I'll update the PR once all requested changes have been made.